src/Integration/WPCOMProxy.php

```diff
@@ -169,36 +169,59 @@ function ( $response, $handler, $request ) {
 					return $response;
 				}
 
+				$data = $response->get_data();
```

Comment:
The `$response` can be one of these types: `WP_REST_Response`, `WP_HTTP_Response`, `WP_Error`, or `mixed`. If `$response` isn't a `WP_REST_Response`, it will throw an error because `get_data` or `set_data` might no be defined.